conn/physic/units.go

```diff
@@ -1552,6 +1739,50 @@ func valueOfUnitString(s string, base prefix) (int64, int, error) {
 	return v, n, nil
 }
 
+// Calcululates the product of two decimals; a and b, keeping the base less than
+// maxInt64. Degrade controls the maximum limit of the precision loss in the
+// product. Returns an error if the product base exceeds an int64 after losing a
+// degrade number of least significant figures. This function is to aid in the
+// multiplication of numbers that combined have more than 18 significant figures
+// each. The minimum limit of significant figures is 9 figures.
+func decimalMul(a, b decimal) (decimal, uint) {
+	if a.base > 18446744073709551609 || b.base > 18446744073709551609 {
```

Comment:
Can you comment where this number comes from?
1<<64 is 18446744073709551616, not 18446744073709551609.
But in practice, anything higher than 1<<63 is game over.

Reply:
Can you add this a a comment?